.github/workflows/spelling.yml

```diff
@@ -0,0 +1,46 @@
+name: Spell checking
+on:
+  pull_request_target:
```

Comment:
For future reference, the [`pull_request_target`](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request_target) event itself is a security concern:

> Warning: The `pull_request_target` event is granted a read/write repository token and can access secrets, even when it is triggered from a fork. Although the workflow runs in the context of the base of the pull request, you should make sure that you do not check out, build, or run untrusted code from the pull request with this event.